app/summary.py

```diff
@@ -0,0 +1,50 @@
+import pickle
+import pandas as pd
+from datetime import datetime
+
+from jjit import search_jjit
+from pracuj import search_pracuj
+
+db_path = 'offers_db'
+
+def prepare_tech_dict():
+
+    offers_db = pickle.load(open(db_path, 'rb'))
+
+    tech_dict = {}
+
+    for offer_list in offers_db['technologies']:
+        for tech in offer_list:
+            if tech in tech_dict:
+                tech_dict[tech] += 1
+            else:
+                tech_dict[tech] = 1
+
+    with open('tech_dict', 'wb') as file:
+        pickle.dump(tech_dict, file)
+
+def search_offers(pracuj_url, jjit_categories, jjit_experience):
+    offers_jjit = search_jjit(jjit_categories, jjit_experience)
```

Comment:
Tutaj znowu powinny być dwie linie przerwy między funkcjami. Jak przeczytasz PEP8 to wszystkie będzie jasne